.storybook/Setup.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Canvas, useFrame } from 'react-three-fiber'
+
+export function Setup({ children }) {
+    return (
+        <Canvas colorManagement shadowMap camera={{ position: [-5, 5, 5] }} pixelRatio={window.devicePixelRatio}>
+            {children}
+            <ambientLight intensity={0.8} />
+            <pointLight intensity={1} color={'ffffff'} position={[0, 6, 0]} />
```

Comment:
The colour is missing a hash (#) you should add a hash, alternatively pointLight has a default colour of white, in theory the colour could be removed altogether.